packages/@tinacms/react-toolbar/src/components/Toolbar.tsx

```diff
@@ -67,13 +69,26 @@ export const Toolbar = () => {
   //   return null
   // }
   // TODO: Form#reset should always exist
-  const reset = form && (form.reset || (() => form.finalForm.reset()))
+
+  const reset = () => {
+    if (form) {
+      if (form.reset) {
+        form.reset()
+      } else if (form.finalForm.reset) {
+        form.finalForm.reset()
+      }
+    }
+  }
```

Comment:
> // TODO: Form#reset should always exist

[This was actually fixed.](https://github.com/tinacms/tinacms/blob/master/packages/@tinacms/forms/src/form.ts#L126-L131) This can be simplified:
```suggestion

  const reset = () => {
    if (form) {
      form.reset()
    }
  }
```